dev/.env.docker-compose

```diff
@@ -1,5 +1,5 @@
 DJANGO_CONFIGURATION=DevelopmentConfiguration
-DJANGO_DATABASE_URL=postgres://postgres:postgres@postgres:5432/django
+DJANGO_DATABASE_URL=postgres://postgres:postgres@postgres:${DOCKER_POSTGRES_PORT}/django
```

Comment:
Could this please default to 5432 so this "just works" for the simple case?

Reply:
yeap, thanks! will do/push the fix